src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -37,22 +37,36 @@ case class AddDatasetMetadataToDeposit(row: Int, entry: (DatasetID, Dataset))(im
    *
    * @return `Success` when all preconditions are met, `Failure` otherwise
    */
-  override def checkPreconditions: Try[Unit] = verifyDataset(row, dataset)
+  override def checkPreconditions: Try[Unit] = {
+    (datasetRowValidations(row, dataset) ++ datasetColumnValidations(row, dataset))
+      .collectResults
+      .map(_ => ())
+  }
 
   override def execute(): Try[Unit] = writeDatasetMetadataXml(row, datasetID, dataset)
 }
 object AddDatasetMetadataToDeposit {
 
-  def verifyDataset(row: Int, dataset: Dataset): Try[Unit] = {
+  def datasetColumnValidations(row: Int, dataset: Dataset): Seq[Try[Unit]] = {
+    import validators._
+    List(
+      checkColumnHasOnlyOneValue(row, dataset, "DDM_CREATED"),
+      checkColumnsHaveAtMostOneRowWithValues(row, dataset, "SF_DOMAIN", "SF_USER", "SF_COLLECTION", "SF_PRESENTATION")
```

Comment:
SF_PRESENTATION is not yet defined at this stage. (maybe it should not even be present?)